charts/ocis/templates/thumbnails/deployment.yaml

```diff
@@ -66,6 +66,13 @@ spec:
             - name: THUMBNAILS_DATA_ENDPOINT
               value: http://{{ .appName }}:9186/thumbnails/data
 
+            {{- if  .Values.services.thumbnails.quota.enabled}}
```

Comment:
I don't see that option documented and wonder if we even need that? If we default maxConcurrencyRequests to 0 and maxFileSize to 50MB, just like the oCIS product, we should be fine.